cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt

```diff
@@ -73,28 +60,23 @@ end
 # Authentication
 ###############################################################################
 
-auth "azure_auth", type: "oauth2" do
-  token_url join(["https://login.microsoftonline.com/",$param_azure_tenant_id,"/oauth2/token"])
-  grant type: "client_credentials" do
-    client_id cred("AZURE_APPLICATION_ID")
-    client_secret cred("AZURE_APPLICATION_KEY")
-    additional_params do {
-      "resource" => "https://management.azure.com/"
-    } end
-  end
+#authenticate with Azure
+credentials "azure_auth" do
+  schemes "oauth2"
+  label "Azure"
+  description "Azure Resource Manager"
+  tags "provider=azure_rm"
 end
 
-auth "log_analytics_auth", type: "oauth2" do
-  token_url join(["https://login.microsoftonline.com/",$param_azure_tenant_id,"/oauth2/token"])
-  grant type: "client_credentials" do
-    client_id cred("AZURE_APPLICATION_ID")
-    client_secret cred("AZURE_APPLICATION_KEY")
-    additional_params do {
-      "resource" => "https://api.loganalytics.io"
-    } end
-  end
+#authenticate with Log Analytics
+credentials "log_analytics_auth" do
+  schemes "oauth2"
+  label "Log Analytics"
+  description "Please select the credentials for Log Analytics"
```

Comment:
Please change the description to **Select the Azure Resource Manager Credential for Log Analytics.**

Reply:
Updated this description.